swagger_spec_validator/validator20.py

```diff
@@ -35,10 +36,14 @@ def validate_spec(spec_json, _spec_url=None):
     """
     validate_json(spec_json, 'schemas/v2.0/schema.json')
 
+    # Dereference all $refs so we don't have to deal with them
+    fix_malformed_model_refs(spec_json)
```

Comment:
Still i believe there is no advantage of going through the flow (L40-42) again after going through them in bravado. How about extracting L40-42 out of the method to a separate `dereference_spec(json)` and it is the responsibility of standalone `ssv` callee to call `dereference_spec` before `validate_spec`. Other approach could be to identify that if the call has come from `bravado` then skip these steps.